run_mode: 0   # 0：速度模式；1：速度位置模式
reduction_ratio: 35  
encoder_lines_num: 2000   # 4倍频后的编码器线数
target_data:
  target_pos_vec: [22.5, 45.0, 90.0, 180.0]    # 目标位置(°)